src/main/java/io/zeebe/monitor/zeebe/kafka/importers/KafkaErrorImporter.java

```diff
@@ -0,0 +1,38 @@
+package io.zeebe.monitor.zeebe.kafka.importers;
```

Comment:
please don't break package structure 
currently it's `zeebe.importers`
so `zeebe.importers.hazelcast` & `zeebe.importers.kafka` I guess?

Reply:
My intention was to keep our code separate until get a decision about contribution. Thats why I kept all of our change in one package though kafkaConfig file outside now.

Anyway I can update package to zeebe.importers.kafka but I don't want to touch on existing package for hazelcast otherwise we may face difficulties during getting new changes from zeebe-simple-monitor.

Reply:
refactored